sycl/doc/extensions/experimental/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -1506,153 +1376,111 @@ any in-flight executions of that same graph if required by the backend.
 This function may only be called if the graph was created with the `updatable`
 property.
 
-Constraints:
+_Constraints:_
 
 * This member function is only available when the `command_graph` state is
   `graph_state::executable`.
 
-Parameters:
+_Throws:_
 
-* `source` - Modifiable graph object used as the source for updating this graph.
-
-Exceptions:
-
-* Throws synchronously with error code `invalid` if `source` contains any node
+* Synchronously with error code `invalid` if `source` contains any node
   which is not one of the following types:
 
 ** `node_type::empty`
 ** `node_type::ext_oneapi_barrier`
 ** `node_type::kernel`
 
-* Throws synchronously with error code `invalid` if the context or device
+* Synchronously with error code `invalid` if the context or device
   associated with `source` does not match that of the `command_graph` being
   updated.
 
-* Throws synchronously with error code `invalid` if
+* Synchronously with error code `invalid` if
   `property::graph::updatable` was not set when the executable graph was
   created.
 
 * If any other exception is thrown the state of the graph nodes is undefined.
-|===
 
-Table {counter: tableNumber}. Member functions of the `command_graph` class for
-queue recording.
-[cols="2a,a"]
-|===
-|Member function|Description
+===== Member functions of the `command_graph` class for queue recording.
 
-|
 [source, c++]
 ----
 void
 begin_recording(queue& recordingQueue,
                 const property_list& propList = {})
 ----
 
-|Synchronously changes the state of `recordingQueue` to the
+_Effects:_ Synchronously changes the state of `recordingQueue` to the
 `queue_state::recording` state. This operation is an error if `recordingQueue`
 is already in the `queue_state::recording` state.
 
-Parameters:
-
-* `recordingQueue` - A `sycl::queue` object to change to the
-  `queue_state::recording` state and start recording commands to the graph
-  instance.
+_Throws:_
 
-* `propList` - Optional parameter for passing properties. Properties for
```

Comment:
We've dropped information about the `propList` parameter , should we have a preconditions section mentioning this parameter the graph-properties section.